main.go

```diff
@@ -70,7 +80,9 @@ var (
 func StartWithOptions(ctx context.Context, version string, so ...ServerOption) (*Server, error) {
 	var err error
 
-	server := &Server{}
+	server := &Server{
+		minMongoLogLvl: LogDebug,
```

Comment:
Just wondering if we should default to Info for normal running.

Reply:
the previous behaviour was "all", so I didn't want to change that